yb-voyager/cmd/importData.go

```diff
@@ -1022,26 +1006,23 @@ func getCopyCommand(table string) string {
 	return "" // no-op
 }
 
-func setProgressAmount(filePath string, progressAmount int64) {
-	log.Debugf("set user.progress_amount=%d for file %q", progressAmount, filePath)
-	s := fmt.Sprintf("%d", progressAmount)
-	err := xattr.Set(filePath, "user.progress_amount", []byte(s))
-	if err != nil {
-		utils.ErrExit("set progress_amount for file %q as %d: %v", filePath, progressAmount, err)
-	}
-}
-
 func getProgressAmount(filePath string) int64 {
-	data, err := xattr.Get(filePath, "user.progress_amount")
-	if err != nil {
-		utils.ErrExit("get extended attribute of file %q: %v", filePath, err)
+	splitName := filepath.Base(filePath)
+	parts := strings.Split(splitName, ".")
+
+	var p int64
+	var err error
+	if dataFileDescriptor.TableRowCount != nil {
+		p, err = strconv.ParseInt(parts[len(parts)-3], 10, 64)
+	} else {
+		p, err = strconv.ParseInt(parts[len(parts)-2], 10, 64)
```

Comment:
Can you please add a comment describing what this if-else block is doing and why?

Reply:
done